install_config/syncing_groups_with_ldap.adoc

```diff
@@ -229,7 +229,7 @@ $ oadm groups prune --sync-config=config.yaml --confirm
 ----
 
 [[sync-examples]]
-== Sync Examples
+== Direct Membership Sync Examples
```

Comment:
This makes sense in the context of "recursive membership" but most people won't even see that.